client/src/components/layout/PlanetTile.js

```diff
@@ -1,11 +1,14 @@
 import React from "react";
+import { Link } from "react-router-dom"
 
 const PlanetTile = ({ id, name, imageUrl }) => {
     return(
-        <div className="cell-small-4 planet-div" key={id}>
-            {name}
-            <img src={imageUrl} width="50px"></img>
-        </div>
+        <Link to={`/${id}`}>
+            <div className="cell-small-3 planet-div" key={id}>
```

Comment:
I didnt catch this earlier, but this element should NOT have a `key` prop. Remove.